kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -271,4 +283,17 @@ public void selectServer(NetFilter.NetFilterContext context) {
             context.initiateConnect(target, filters);
         }
     }
+
+    @Sharable
+    private static class ErrorHandler extends ChannelInboundHandlerAdapter {
```

Comment:
LoggingErrorHandler perhaps?

Reply:
I'll also include the Inbound-ness in the name